### PR TITLE
stress-ng: update to 0.14.03

### PR DESCRIPTION
**Pull request by guijan**
Signed-off-by: Guilherme Janczak <guilherme.janczak@yandex.com>

Maintainer: @commodo
Compile tested: x64, latest snapshot SDK as of today

### CODE REVIEW

**Comment by commodo** [comment]
is this build error new?  

https://github.com/openwrt/packages/runs/7495316112?check_suite_focus=true

i am not sure if this was present;
tbh: the ARC toolchains have always been problematic

i'll need to see if we can somehow fix it (if it wasn't broken before);

**Comment by commodo** [comment]
other than ARC, rest is LGTM

**Comment by guijan** [comment]
It's new, the old version succeeded: https://github.com/openwrt/packages/runs/4019861942?check_suite_focus=true

Odd that this fails on ARC, the relevant Linux headers seem to be arch-independent...

**Comment by neheb** [comment]
Probably a result of glibc update.

**Comment by guijan** [comment]
Sorry, I didn't test adding the patch, I'll do it properly.

File: utils/stress-ng/patches/010-dont_define_fortify.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+--- a/stress-ng.h
++++ b/stress-ng.h
+@@ -34,10 +34,6 @@
+ #ifndef _LARGEFILE64_SOURCE
+ #define _LARGEFILE64_SOURCE
+ #endif
+-#if !defined(__PCC__) && 	\
+-    !defined(__TINYC__)
+-#define _FORTIFY_SOURCE 2
+-#endif
```
Comment (commodo) [inline utils/stress-ng/patches/010-dont_define_fortify.patch:10]: this patch is fine, but i don't see it fixing errors;
it's fixing warnings;

if possible i would send it upstream to have it discussed with them;
they are reasonably quick;

i would not carry patches in our tree if they don't fix anything more grave;

**Pull request merged**